package-lock.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "octokit",
   "version": "0.0.0-development",
-  "lockfileVersion": 2,
```

Comment:
Can you regenerate this so it is lockfileVersion 2
Version 3 isn't compatible with older versions of npm